### PR TITLE
SQL queries for train predict and label data

### PR DESCRIPTION
**Pull request by riteshkarval**

### CODE REVIEW

File: samples/train_metrics.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "train_metrics": {
+        "mse": 580148.0536563194,
+        "mae": 567.6535461416039,
+        "rmse": 761.6745063715337
```
Comment (songole) [inline samples/train_metrics.json:7]: How do we take multidimensional metrics?

File: insurance/README.md [hunk 2]
```diff
@@ -70,10 +70,12 @@
 1. Name : insurance-training-data 
    - This is DKube local dataset, created by the training/retraining pipeline above
 2. Type : csv
-3. Version: v2
-4. Add transformer script: https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance/transform-data.py
-5. Note: Download the script in your setup and then add it by browsing.
-6. Save training data.
+3. If training data source is SQL
+      - Add query: `select * from insurance` (the table name can be different)
+4. If training Version: v2
```
Comment (songole) [inline insurance/README.md]: This line doesn't fit with the flow. Are you saying, if v2 Add Transformer?

File: insurance/README.md [hunk 3]
```diff
@@ -70,10 +70,12 @@
 1. Name : insurance-training-data 
    - This is DKube local dataset, created by the training/retraining pipeline above
 2. Type : csv
-3. Version: v2
-4. Add transformer script: https://github.com/oneconvergence/dkube-examples/tree/monitoring/insurance/transform-data.py
-5. Note: Download the script in your setup and then add it by browsing.
-6. Save training data.
+3. If training data source is SQL
```
Comment (songole) [inline insurance/README.md:75]: Can we move this under 2: Type: csv as a sub bullet

File: samples/train_metrics.json (hunk 1 above)
Comment (riteshkarval) [reply samples/train_metrics.json:7]: added sample in the file

File: insurance/README.md (hunk 2 above)
Comment (riteshkarval) [reply insurance/README.md]: It's moved up.

File: insurance/README.md (hunk 3 above)
Comment (riteshkarval) [reply insurance/README.md:75]: added the changes

**Pull request merged**